--- /home/kristianf/Makefile	2013-09-22 21:35:20.000000000 +0200
+++ Makefile	2013-09-22 21:53:14.000000000 +0200
@@ -29,4 +29,14 @@
 debian-tag:
 	git tag -s msva-perl_debian/$(DEBIAN_VERSION) -m "tagging msva-perl debian packaging version $(DEBIAN_VERSION)"
 
+install: all
+	install -m 0444 -D Crypt/Monkeysphere/MSVA.pm ##PERL##/Crypt/Monkeysphere/MSVA.pm
+	install -m 0444 Crypt/Monkeysphere/*.pm ##PERL##/Crypt/Monkeysphere
+	install -m 0444 -D Crypt/Monkeysphere/MSVA/MarginalUI.pm ##PERL##/Crypt/Monkeysphere/MSVA/MarginalUI.pm
+	install -m 0444 Crypt/Monkeysphere/MSVA/*.pm ##PERL##/Crypt/Monkeysphere/MSVA
+	install -m 0444 -D Net/Server/MSVA.pm ##PERL##/Net/Server/MSVA.pm
+	install -m 0555 -D msva-perl ${ED}/usr/bin/msva-perl
+	install -m 0555 -D msva-query-agent ${ED}/usr/bin/msva-query-agent
+
 .PHONY: upstream-tag debian-package debian-tag all clean
+
