diff -r 984d935e1ab9 src/screenshot-filename-builder.c
--- a/src/screenshot-filename-builder.c	Mon Dec 01 13:32:02 2014 +0100
+++ b/src/screenshot-filename-builder.c	Mon Dec 01 13:33:00 2014 +0100
@@ -132,7 +132,7 @@
       GDateTime *d;
 
       d = g_date_time_new_now_local ();
-      origin = g_date_time_format (d, "%Y-%m-%d %H:%M:%S");
+      origin = g_date_time_format (d, "%Y-%m-%d-%H-%M-%S");
       g_date_time_unref (d);
     }
   else
