diff -r 5061671dbcae src/polly/gui/twitter/account.py
--- a/src/polly/gui/twitter/account.py	Wed Jun 11 23:55:49 2014 +0200
+++ b/src/polly/gui/twitter/account.py	Wed Jun 11 23:57:23 2014 +0200
@@ -483,7 +483,7 @@
                     if id < 0:
                         raise ValueError
 
-                    subtoken_secret = keyring.get_password(self.name, subtoken)
+                    subtoken_secret = keyring.get_password(self.name.encode(ENCODING), subtoken)
 
                     if subtoken_secret is None:
                         raise ValueError
